### ISSUE TITLE
The best validation accuracy is not the best.

### ISSUE THREAD

**Issue by JaneShenYY** [barronalex/Dynamic-Memory-Networks-in-TensorFlow#14]
At the end of the training, the best validation accuracy reported is not the best. For example, task 7, the best validation accuracy reported is 0.975, while there are 0.977, 0.978 for the validation accuracy in different epochs.
